README.md

```diff
@@ -424,7 +424,7 @@ Variable | Description | Default value
 `JICOFO_AUTH_PASSWORD` | XMPP password for Jicofo client connections | passw0rd
 `JVB_AUTH_USER` | XMPP user for JVB MUC client connections | jvb
 `JVB_AUTH_PASSWORD` | XMPP password for JVB MUC client connections | passw0rd
-`JVB_STUN_SERVERS` | STUN servers used to discover the server's public IP | stun.l.google.com:19302, stun1.l.google.com:19302, stun2.l.google.com:19302
+`JVB_STUN_SERVERS` | STUN servers used to discover the server's public IP | none
```

Comment:
The default is already our own server: https://github.com/jitsi/docker-jitsi-meet/blob/4e2cec6b38610ef150576f52bb3f27b0b93206a0/env.example#L210

Reply:
Ping?

Reply:
I test `telnet meet-jit-si-turnrelay.jitsi.net 443` is working. by the way, can you help me to explain what is the benefit of using jitsi stun server. is it faster than `stun.l.google..`?

Reply:
It’s not a Google server. You are free to use whichever you want.